packages/tc-api-graphql/lib/get-augmented-schema.js

```diff
@@ -34,8 +34,18 @@ const getDocumentResolvers = () => {
 	return typeResolvers;
 };
 
-const getAugmentedSchema = ({ documentStore }) => {
+const getAugmentedSchema = ({ documentStore, options }) => {
+	const resolvers = documentStore ? getDocumentResolvers() : {};
 	const typeDefs = getGraphqlDefs();
+
+	if (options.schema) {
+		typeDefs.push(options.schema);
```

Comment:
```suggestion
		typeDefs.push(...options.typeDefs);
```
I think you need to spread if it's an array?

Also prefer typeDefs to schema as typeDefs is what the graphql libraries use